client/src/SubmitBidForm.js

```diff
@@ -1,14 +1,15 @@
 import React, { useState, useEffect } from "react";
 import { useNavigate, useParams } from "react-router-dom";
 import { get, post } from "./TenderClient";
+import "./SubmitBidForm.css";
 
 const SubmitBidForm = () => {
 	const { tenderId } = useParams();
 	const [coverLetter, setCoverLetter] = useState("");
 	const [proposedDuration, setProposedDuration] = useState("");
 	const [proposedBudget, setProposedBudget] = useState("");
-	const [errors, setErrors] = useState([]);
-	const [bidError, setBidError] = useState("");
+	const [registerStatus, setRegisterStatus] = useState("");
```

Comment:
registerStatus is a bad name for this variable, and it's also unnecessary. The `errors` variable in the next line is enough